indico/modules/events/editing/service.py

```diff
@@ -149,7 +150,12 @@ def service_handle_new_editable(editable, user):
         )
         resp = requests.put(_build_url(editable.event, path), headers=_get_headers(editable.event), json=data)
         resp.raise_for_status()
-    except requests.RequestException as exc:
+        resp = ServiceCreateEditableResultSchema().load(resp.json()) if resp.text else {}
+
+        if resp.get('ready_for_review', False):
```

Comment:
```suggestion
        if resp.get('ready_for_review'):
```

the default `None` is falsy as well